# Methods

Qiime2 (version 2019.10) was used to analyze the 16S data between the treatment groups (probiotic, Live Salmonella and Probiotic + Live Salmonella). Sequences were demultiplexed using the Qiime demux emp-paired function and denoised using the Qiime 2 plugin dada2.  The number of good quality reads for taxonomic assignment ranged from 27,272 to 60368 reads.  The pre-made greengenes database (version 13.8 at the 99% operational taxonomic units (OTUs) for the region (515F/806R)) was used to classify each of the reads using Qiime's feature-classifier function.  Alpha diversity, beta diversity analysis and unweighted unifrac pcoa were calculated using Qiime2's built in functions. Qiimes gneiss plugin was used to explore taxonomic balances and taxonomic group differences between treatment groups.  The ols-regression summary indicated the model used (Treatment+Unknown) was a good fit for the data with small residuals. A large unknown factor was noted to account for 40% of the variation but this variation was orthogonal to the variation that can discriminate between treatment groups and can be safely ignored. For a more thorough description of our step-by-step methods, please refer to the github repo at https://github.com/ISUgenomics/MelhaMellata.
