1. Processing raw reads: Raw read counts were obtained after separating them to various samples and quality check was performed. First tab will provide you counts for every sample. You can also visualize the quality of all the reads by clicking on the `Interactive Sample Detail`  tab at the top.			
	- demultiplexed (data)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/uk7hm5my8p1izus/combined-demux.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/uk7hm5my8p1izus/combined-demux.qza?dl=0))
	- demultiplexed (stats)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/3anuigjt3vlk27q/combined-demux.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/3anuigjt3vlk27q/combined-demux.qzv?dl=0))
2. Finding sequence variants: to cluster/collapse all similar reads, overlooking the sequence errors, [DADA2](https://www.nature.com/articles/nmeth.3869) was used. Once processed, it will give us representative sequences (collapsed reads and visualization file). Again click on the `Interactive Sample Detail`  tab to see more information about each sample. Choose different conditions to see the differneces in treatment.			
	- Table dada2 (plots)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/9w5y8a4hvxymes7/table-dada2.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/9w5y8a4hvxymes7/table-dada2.qzv?dl=0))
	- Table dada2 (data)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/4xr41so25jm3gg2/table-dada2.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/4xr41so25jm3gg2/table-dada2.qza?dl=0))
	- Representative Sequences (data)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/fybw6b4pwatmwlo/rep-seqs-dada2.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/fybw6b4pwatmwlo/rep-seqs-dada2.qza?dl=0))
	- Representative Sequences (plots)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/chud97gpsnhlo6d/rep-seqs-dada2.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/chud97gpsnhlo6d/rep-seqs-dada2.qzv?dl=0))
	- Aligned Rep sequences (masked)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/33umiifottm9fgf/masked-aligned-rep-seqs.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/33umiifottm9fgf/masked-aligned-rep-seqs.qza?dl=0))
	- Aligned Rep sequences	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/k7ev0h5313nx7z5/aligned-rep-seqs.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/k7ev0h5313nx7z5/aligned-rep-seqs.qza?dl=0))
	- Stats dada2 (data)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/z3qsdmmwcisu71t/stats-dada2.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/z3qsdmmwcisu71t/stats-dada2.qza?dl=0))
3. Taxonomical analysis: After collapsing the reads, taxonomy was assigned to the representative sequences in our data. We used [silva](https://www.arb-silva.de/) 16s database (release 128, 99% OTUs), custom trained for the region 515f to 926r.  Once we know the OTU's for rep sequences, we can then apply this to all of the other reads in our dataset to obtain taxa bar plots. This interactive visualization will let you see the composition of various organisms, at different levels. You can export the abundances at the level of your choice to excel sheet format, by clicking on `csv` at the top.			
	- Rooted tree	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/g8a4gk364rk7bk9/rooted-tree.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/g8a4gk364rk7bk9/rooted-tree.qza?dl=0))
	- Unrooted tree	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/5t82n9pijyuogjv/unrooted-tree.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/5t82n9pijyuogjv/unrooted-tree.qza?dl=0))
	- Taxonomy (data)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/v5ee8xha6hu5x3f/taxonomy.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/v5ee8xha6hu5x3f/taxonomy.qza?dl=0))
	- Taxonomy (plots)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/f0z8ny6lu9gpxvl/taxa-bar-plots.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/f0z8ny6lu9gpxvl/taxa-bar-plots.qzv?dl=0))
4. Alpha diversity			
	- Shannon’s diversity index (a quantitative measure of community richness)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/yva7j459ut2j0kn/shannon_vector.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/yva7j459ut2j0kn/shannon_vector.qza?dl=0))
	- Observed OTUs (a qualitative measure of community richness)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/ks1q61d32mygc58/observed_otus_vector.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/ks1q61d32mygc58/observed_otus_vector.qza?dl=0))
	- Faith’s Phylogenetic Diversity (a qualitiative measure of community richness that incorporates phylogenetic relationships between the features)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/dygvwfl7r698s69/faith_pd_vector.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/dygvwfl7r698s69/faith_pd_vector.qza?dl=0))
	- Evenness (or Pielou’s Evenness; a measure of community evenness)	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/n0r4cchwkap8g3p/evenness_vector.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/n0r4cchwkap8g3p/evenness_vector.qza?dl=0))
5. Beta diversity			
	a. Jaccard distance (a qualitative measure of community dissimilarity) 		
		 * Jaccard distance matrix	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/yiit3ftsjzm3rzt/jaccard_distance_matrix.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/yiit3ftsjzm3rzt/jaccard_distance_matrix.qza?dl=0))
		 * jaccard pcoa results	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/8gzzco43itov4ae/jaccard_pcoa_results.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/8gzzco43itov4ae/jaccard_pcoa_results.qza?dl=0))
		 * jaccard emperor results	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/8zpud0dzv8h3jos/jaccard_emperor.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/8zpud0dzv8h3jos/jaccard_emperor.qzv?dl=0))
	b. Bray-Curtis distance (a quantitative measure of community dissimilarity) 		
		 * bray curtis distance matrix	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/9z88fpw1b7tiao1/bray_curtis_distance_matrix.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/9z88fpw1b7tiao1/bray_curtis_distance_matrix.qza?dl=0))
		 * bray curtis pcoa results	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/hrk1crv7fsom3gh/bray_curtis_pcoa_results.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/hrk1crv7fsom3gh/bray_curtis_pcoa_results.qza?dl=0))
	c. unweighted UniFrac distance (a qualitative measure of community dissimilarity that incorporates phylogenetic relationships between the features) 		
		 * unweighted unifrac distance matrix	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/vgal7w7dhd3xm3x/unweighted_unifrac_distance_matrix.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/vgal7w7dhd3xm3x/unweighted_unifrac_distance_matrix.qza?dl=0))
		 * unweighted unifrac pcoa results	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/pfnn1l6wc4a521y/unweighted_unifrac_pcoa_results.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/pfnn1l6wc4a521y/unweighted_unifrac_pcoa_results.qza?dl=0))
	d. weighted UniFrac distance (a quantitative measure of community dissimilarity that incorporates phylogenetic relationships between the features) 		
		 * weighted unifrac distance matrix	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/x1ev4w3hwx2b7oz/weighted_unifrac_distance_matrix.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/x1ev4w3hwx2b7oz/weighted_unifrac_distance_matrix.qza?dl=0))
		 * weighted unifrac pcoa results	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/atc5b6vj77w2xpi/weighted_unifrac_pcoa_results.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/atc5b6vj77w2xpi/weighted_unifrac_pcoa_results.qza?dl=0))
		 * weighted unifrac emperor results	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/trslgtflwqpiv3j/weighted_unifrac_emperor.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/trslgtflwqpiv3j/weighted_unifrac_emperor.qzv?dl=0))
6. Microbial composition:  Associations between discrete metadata categories and alpha diversity was done based on Faith Phylogenetic Diversity (a measure of community richness) and evenness metrics.			
	- faith-pd-group-significance	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/k4d6ipecmlw5ep1/faith-pd-group-significance.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/k4d6ipecmlw5ep1/faith-pd-group-significance.qzv?dl=0))
	- evenness-group-significance	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/736xejkkvuhl67s/evenness-group-significance.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/736xejkkvuhl67s/evenness-group-significance.qzv?dl=0))
7. PCoA analysis: unweighted UniFrac and Bray-Curtis results can also be explored as principal co-ordinates (PCoA) plots. Plots were generated using Emperor tool.			
	- unweighted-unifrac-emperor	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/0jaqvh4c9ar1i7w/unweighted_unifrac_emperor.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/0jaqvh4c9ar1i7w/unweighted_unifrac_emperor.qzv?dl=0))
	- bray-curtis-emperor	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/01kufmxplaxnzob/bray_curtis_emperor.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/01kufmxplaxnzob/bray_curtis_emperor.qzv?dl=0))
8. Rareifaction			
	Alpha rarefaction curves	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/tqsdyvbv3os9boy/alpha-rarefaction.qzv%3Fdl%3D1) [download](https://www.dropbox.com/s/tqsdyvbv3os9boy/alpha-rarefaction.qzv?dl=0))
	Rarefied table	([view](https://view.qiime2.org/peek/?src=https%3A//dl.dropbox.com/s/51cn69wmrgozea9/rarefied_table.qza%3Fdl%3D1) [download](https://www.dropbox.com/s/51cn69wmrgozea9/rarefied_table.qza?dl=0))
